horace_core/admin/check_horace_mex.m

```diff
@@ -1,17 +1,29 @@
-function [rez, n_errors, can_use_mex_4_combine] = check_horace_mex()
+function [rez, n_errors, can_use_mex_4_combine,can_use_custom_MPI] = check_horace_mex()
```

Comment:
What is `custom` here? What makes it `custom`? Do you mean system?

Reply:
custom means created by us, not provided by Matlab.